zebra-chain/src/sapling/arbitrary.rs

```diff
@@ -125,7 +125,10 @@ impl Arbitrary for tree::Root {
         (vec(any::<u8>(), 64))
             .prop_map(|bytes| {
                 let bytes = bytes.try_into().expect("vec is the correct length");
-                jubjub::Fq::from_bytes_wide(&bytes).to_bytes().into()
+                jubjub::Fq::from_bytes_wide(&bytes)
+                    .to_bytes()
+                    .try_into()
+                    .expect("bytes are always valid")
```

Comment:
Question: these bytes are random, have we already checked they are valid? Or could this panic?

If some bytes are invalid, we can use `prop_filter_map` and `ok` to skip it:
https://docs.rs/proptest/latest/proptest/strategy/trait.Strategy.html#method.prop_filter_map
https://doc.rust-lang.org/std/result/enum.Result.html#method.ok

Reply:
There's a fix below that will make this not required anymore